project/SparkBuild.scala

```diff
@@ -600,7 +600,7 @@ object DockerIntegrationTests {
   lazy val settings = Seq(
     dependencyOverrides += "com.google.guava" % "guava" % "18.0",
     resolvers += "DB2" at "https://app.camunda.com/nexus/content/repositories/public/",
-    libraryDependencies += "com.oracle" % "ojdbc6" % "11.2.0.1.0" from "https://app.camunda.com/nexus/content/repositories/public/com/oracle/ojdbc6/11.2.0.1.0/ojdbc6-11.2.0.1.0.jar" // scalastyle:ignore
+    libraryDependencies += "com.oracle" % "ojdbc6" % "11.2.0.3" from "https://app.camunda.com/nexus/content/repositories/public/com/oracle/ojdbc6/11.2.0.3/ojdbc6-11.2.0.3.jar" // scalastyle:ignore
```

Comment:
The previously referenced version no longer exists in jfrog, so I updated to the closest similar version that was available. https://camunda.jfrog.io/ui/native/public/com/oracle/ojdbc6

Reply:
cc @sarutak FYI

Reply:
BTW, is it something we can backport up to branch-3.1? If not, let's separate these changes into a separate PR so that can be only merged to master or other branches.

Reply:
Hmm, I'm confused about this part. Do we need this dependency setting here?
`pom.xml` has the setting for `ojdbc8`.

Reply:
I'm going to pull this into another PR for the sake of simplifying any backporting. I will update with a link shortly.